src/openms/include/OpenMS/ANALYSIS/OPENSWATH/MRMFeatureFilter.h

```diff
@@ -153,6 +222,57 @@ namespace OpenMS
       bool & key_exists
     ) const;
 
+    /**
+      @brief Updates the metaValue ranges based on the value given
+
+      @param[in] component component of the numerator
+      @param[in] meta_value_key Name of the metaValue
+      @param[in, out] meta_value_l Lower bound (inclusive) for the metaValue range
+      @param[in, out] meta_value_u Upper bound (inclusive) for the metaValue range
+      @param[out] key_exists true if the given key is found, false otherwise
+    */
+    void updateMetaValue(
+      const Feature & component,
```

Comment:
coding convention: `Feature& component....`

Reply:
You don't like the default MSVC linting I see?